scripts/install.sh

```diff
@@ -5,32 +5,44 @@ if [ "$(id -u)" != "0" ]; then
    exit 1
 fi
 
+ARCH=$(uname -m)
+ARCH_NAME=""
+if [ "$ARCH" = "aarch64" ]; then
+  ARCH_NAME="linuxarm64"
+elif [ "$ARCH" = "aarch32" ]; then
+  ARCH_NAME="linuxarm32"
+elif [ "$ARCH" = "x86_64" ]; then
+  ARCH_NAME="linuxx64"
+else
+  if [ "$#" -ne 1 ]; then
+      echo "Can't determine current arch; please provide it (one of):"
+      echo ""
+      echo "- linuxarm32 (32-bit Linux ARM)"
+      echo "- linuxarm64 (64-bit Linux ARM)"
+      echo "- linuxx64   (64-bit Linux)"
+      exit 1
+  else
+    echo "Can't detect arch (got $ARCH) -- using user-provided $1"
+    ARCH_NAME=$1
+  fi
+fi
+
 echo "This is the installation script for PhotonVision."
+echo "Installing for platform $ARCH_NAME"
 
 echo "Installing curl..."
-apt-get install --yes curl
+apt-get install curl
 echo "curl installation complete."
 
 echo "Installing avahi-daemon..."
-apt-get install --yes avahi-daemon
+apt-get install avahi-daemon
 echo "avahi-daemon installation complete."
 
-echo "Installing cpufrequtils..."
-apt-get install --yes cpufrequtils
-echo "cpufrequtils installation complete."
-
-echo "Setting cpufrequtils to performance mode"
-if [ -f /etc/default/cpufrequtils ]; then
-    sed -i -e 's/^#\?GOVERNOR=.*$/GOVERNOR=performance/' /etc/default/cpufrequtils
-else
-    echo 'GOVERNOR=performance' > /etc/default/cpufrequtils
-fi
-
 echo "Installing the JDK..."
 if [ $(dpkg-query -W -f='${Status}' openjdk-11-jdk-headless 2>/dev/null | grep -c "ok installed") -eq 0 ];
 then
-   apt-get update
-   apt-get install --yes openjdk-11-jdk-headless
+   apt update
+   apt-get install openjdk-11-jdk-headless;
```

Comment:
Shouldn't this be jdk 17 now?

Reply:
Not required yet, we still target Java 11 language support. And I don't want to change it coz we don't have time to fully test Java 17 right now.